gecoscc/templates/settings.jinja2

```diff
@@ -70,10 +70,10 @@
                       }
                     }
                 }
-                {% elif setting.type == "Mimetypes" %}
```

Comment:
These fields have been renamed, and you are not pulling latests changes. Please, update your local copy before a new Pull Request.